### ISSUE TITLE
"Restore Purchase" does not do anything.

### ISSUE THREAD

**Issue by jeppsson** [ItsCalebJones/SpaceLaunchNow-Android#195]
**Describe the bug**
"Restore Purchase" does not so anything. 

**To Reproduce**
1. Go to "Become a Supporter" 
2. Click on "Restore Purchase" in overflow menu.

**Expected behavior**
My purchase should be restored.

**Smartphone (please complete the following information):**
 - Device: Pixel 6a
 - OS: Android 1
 - Version

**Additional context**
Code has been commented out.

**Comment by ItsCalebJones** [comment]
Hi there, 

I just tested it on my Pixel 6 Pro on, app version 3.14.1 and OS 13.

My purchase was restored on first install - its true that the 'Restore Purchase' button is not functional because the flow is now async and automatic. There's likely either some bug not being handled or you may be using a different email address.

**Comment by jeppsson** [comment]
Hi!

Thanks for the quick answer.

Uninstalled and installed again. And now it works. Sorry, should have tried that first.

Got a new Pixel. Did a transfer with cable from my old Pixel. I have a bunch of accounts. I registered all accounts on new device. 
I think at first the account were I purchased the app was not the selected on in Google Play. I later changed to correct account and cleared the data in app. But still it didn't restore.

Anyway. After uninstall and reinstall it works now.

Thanks for a great app!
